app/scripts/lib/origin-check.js

```diff
@@ -77,7 +80,11 @@ define([
         .fail(function (err) {
           if (/Timed out/.test(String(err))) {
             // a timeout is A-OK, no allowed origins are listening.
+            Raven.captureException(new Error('Origin check timed out'));
+
             return null;
```

Comment:
from irc: let's fix that in the later train, for now just adjusting the timeout.